src/server/nyplApiClient/index.ts

```diff
@@ -0,0 +1,107 @@
+import NyplApiClient from "@nypl/nypl-data-api-client"
+import aws from "aws-sdk"
+
+interface KMSCache {
+  clients: string[]
+  clientId: string
+  clientSecret: string
+}
+
+const config: any = {
```

Comment:
Eventually this should be moved to a separate config file.

Reply:
Just moved it to a config file. Let me know if this naming/placement makes sense.